apps/src/feedback.js

```diff
@@ -73,7 +73,7 @@ module.exports = FeedbackUtils;
  */
 
 /**
- * @typedef {{ err, lineNumber: number}} ExecutionError
+ * @typedef {{err, lineNumber: number}} ExecutionError
```

Comment:
nit: can we revert this as well (so the diff stays scoped to the relevant files?)

Reply:
Removed in [b2320b2](https://github.com/code-dot-org/code-dot-org/pull/64770/commits/b2320b29c783ff28f4d5a53dfcfabd6c8c699a32). 👍